frontend/src/features/workspace/WorkspacePage.tsx

```diff
@@ -108,6 +111,24 @@ export const WorkspacePage = (): JSX.Element => {
     createFormModalDisclosure,
     whatsNewFeatureDrawerDisclosure,
   } = useWorkspaceForms()
+  const { updateUserLastSeenFeatureUpdateDateMutation } = useUserMutations()
+  const { user, isLoading: isUserLoading } = useUser()
+  const [shouldShowWhatsNewNotification, setShouldShowWhatsNewNotification] =
+    useState(false)
+  const shouldShowFeatureUpdateNotification =
+    shouldShowLatestFeatureUpdateNotification({
+      user,
+      isLoading: isUserLoading,
+    })
+
+  useEffect(() => {
+    setShouldShowWhatsNewNotification(shouldShowFeatureUpdateNotification)
+  }, [shouldShowFeatureUpdateNotification])
```

Comment:
feels a little weird to combine an util fn, effect, and use state. Can't we just memoize this?
also feels a little weird to pass in loading states to the utility function.

suggest just memoizing:
```tsx
const shouldShowFeatureUpdateNotification = useMemo(() => {
  if (isUserLoading || !user) return false
  return getShowLatestFeatureUpdateNotification(user)
}, [user, isUserLoading, getShowLatestFeatureUpdateNotification])
```